cluster-sim/cluster_sim/fake_server.py

```diff
@@ -44,11 +44,11 @@
 # Simulated duration, in seconds, from the time a server shutdown is issued
 # until it's stopped. When simulating a shutdown, it will always take at
 # least this long.
-MIN_SHUTDOWN_DURATION = 10
+MIN_SHUTDOWN_DURATION = 5  # 10
```

Comment:
Then remove the # 20.

But tbh if you are trying to uncover a simulator issue, that is difficult to find, changing other timings are the same time seems to be a way to make the problem harder, not easier.